### DISCUSSION TITLE
kerbros token based authentication support for kafka scaler

### DISCUSSION THREAD

**Discussion by chinnasamyb** [kedacore/keda#1518] [Q&A / Need Help]
We are using ESP enabled HDInsight Kafka cluster in Azure cloud.  We have  kafka consumers deployed to Azure kubernetes cluster (AKS).   We tried to use  kafka scaler to scale kafka consumer pods.  Kafka scaler is not able to connect to ESP enabled HDInsight kafka cluster. Looks like kafka scaler does not support kerbros tokens  yet. Wondering if there are  any plans to add kerbros token support for kafka scaler in the near future?

**Comment by tomkerkhove** [comment]
We have not had this request yet but happy to add it to the backlog. Would you mind opening a new issue to track this with relevant information please?

We do welcome contributions as well so if you are interested in contributing this feature, feel free to let us know!

**Comment by chinnasamyb** [comment]
Thanks for quick response
I created the issue  https://github.com/kedacore/keda/issues/1521.
I am new to golang. However I tinkered around using sarama kerbros module with kafka scaler  to try connecting to kafka with no success.  So I am not sure how much I can contribute to this effort

Thanks for your consideration.

Bala
